src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball firstBall = new Lottery().getRandomBall();
+        Ball secondBall = new Lottery().getRandomBall();
+        Ball thirdBall = new Lottery().getRandomBall();
+        System.out.println(firstBall.toString());
```

Comment:
Use the shorter syntax 'System.out.println(firstBall);' instead of calling .toString() method explicitly. The toString() method will be called automatically when printing an object.